mahta/CSS/CSS responsive design/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<head>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+ <div class="Title">
```

Comment:
Please don't use Upperfirst classnames. Stick to either camelCase or better, kebab-case.